openedx/core/djangoapps/content_tagging/rules.py

```diff
@@ -101,5 +235,5 @@ def can_change_taxonomy_tag(user: UserType, tag: oel_tagging.Tag | None = None)
 
 # This perms are used in the tagging rest api from openedx_tagging that is exposed in the CMS. They are overridden here
 # to include Organization and objects permissions.
-rules.set_perm("oel_tagging.change_objecttag_taxonomy", can_change_object_tag_taxonomy)
+rules.set_perm("oel_tagging.change_objecttag_taxonomy", can_view_taxonomy)
 rules.set_perm("oel_tagging.change_objecttag_objectid", can_change_object_tag_objectid)
```

Comment:
And I believe that these requirements:

> **Who can view ObjectTags**
> 16. Any user who can view a content object (unit, component, or library component) in Studio can view that content object's tags.
> 17. Any user who can not view a content object (unit, component, or library component) in Studio can not view that content object's tags.

mean we need an additional rule like this which checks the `object_id` on view?

```
rules.set_perm("oel_tagging.view_objecttag_objectid", can_view_object_tag_objectid)
```

Reply:
I missed that! Thank you!